corehq/apps/app_manager/templates/app_manager/partials/releases.html

```diff
@@ -100,12 +100,14 @@ <h4 class="modal-title">{% trans "CommCareHQ looks better on desktop!" %}</h4>
                         <a style="cursor: pointer" id="send-mobile-reminder-button">Click here</a> to send yourself an email with a login link to use when you’re back on a full-size browser.
                         {% endblocktrans %}
                     </p>
+                    <img src="{% static 'hqwebapp/images/yikes_face.png' %}"
```

Comment:
Normally I'd say this should be an svg so it'll scale well (@djmore9 it's helpful to link to the noun project so we can get the svg instead of the png) but given that it'll only be seen on small devices I don't think it's a big deal.

Reply:
Totally agree, also not sending down a binary would probably be nicer.

That can be an update later though